panel/pane/alert.py

```diff
@@ -24,7 +24,7 @@ class Alert(Markdown):
 
     alert_type = param.ObjectSelector("primary", objects=ALERT_TYPES)
 
-    priority = 0
+    priority: float | bool | None = 0
```

Comment:
I used the type `float | bool | None` as it was already used in the `PaneBase` class.